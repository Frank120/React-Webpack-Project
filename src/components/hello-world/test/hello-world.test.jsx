import React from 'react';
import renderer from 'react-test-renderer';
import HelloWorld from '../component';

it('renders correctly', () => {
  const tree = renderer.create(
    <HelloWorld />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
