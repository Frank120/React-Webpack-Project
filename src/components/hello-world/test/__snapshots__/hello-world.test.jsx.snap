// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="hello-world"
>
  Hello World!!!!! (from the full lifecycle component)
  <img
    alt="logo"
    className="logo-img"
    src="react-logo.png"
  />
  <span
    className="icon-icon_B039"
  />
</div>
`;
